packages/components/pie-radio-group/test/component/pie-radio-group-new.spec.ts

```diff
@@ -0,0 +1,349 @@
+import { test, expect } from '@playwright/test';
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+
+// The structure of this object reflects the structure of our test Story
+const selectors = {
+    button1: 'btn-1',
+    radioGroup1: {
+        self: 'radio-group-1',
+        radios: {
+            1: 'radio-1',
+            2: 'radio-2',
+            3: 'radio-3',
+            4: 'radio-4',
+            5: 'radio-5',
+        },
+    },
+    button2: 'btn-2',
+    button3: 'btn-3',
+    radioGroup2: {
+        self: 'radio-group-2',
+        radios: {
+            1: 'radio-6',
+            2: 'radio-7',
+            3: 'radio-8',
+            4: 'radio-9',
+            5: 'radio-10',
+        },
+    },
+    button4: 'btn-4',
+};
+
+test.describe('PieRadioGroup - Component tests new', () => {
+    let pageObject;
```

Comment:
Not sure on your thoughts, but I think we could possibly change this from `pageObject` to something like, `pieRadioGroupPage` or `radioGroupPage` as then when you look at the load functions below, I think it is clearer what is trying to be done.

e.g. `await radioGroupPage.load()`

Reply:
We can also totally address this in a follow up PR.

Reply:
cheers mate! happy to merge in a follow up if that's ok :)